tools/topology/topology2/platform/intel/sdw-amp-feedback.conf

```diff
@@ -0,0 +1,64 @@
+Object.Dai {
+	ALH.30 {
+		id 		$SDW_AMP_IN_BE_ID
+		direction	"capture"
+		name		$SDW_SPK_IN_STREAM
+		default_hw_conf_id	1
+		rate			48000
+		channels		2
+
+		Object.Base.hw_config."ALH515" {
```

Comment:
is the 515 here intentional? I thought these were removed

Reply:
> is the 515 here intentional? I thought these were removed

No. I forgot to change the index in hw_config. However, somehow it still works.